web-ui/src/main/resources/catalog/components/viewer/owscontext/OwsContextService.js

```diff
@@ -278,6 +327,8 @@
                   loadingLayer.set("bgLayer", true);
                 }
 
+                setLayerAttribution(loadingLayer, layerAttributionArray);
```

Comment:
Why are you trying to set the attribution on the loading layer?

Reply:
Moved the setLayerAttribution function to the location that you suggested and it now shows the attribution for WMTS maps.
Thank you for the suggestions.